tests/robottelo/test_decorators.py

```diff
@@ -36,20 +39,37 @@ def test_not_smoke(self):
 class BzBugIsOpenTestCase(TestCase):
     """Tests for :func:`robottelo.decorators.bz_bug_is_open`."""
     # (protected-access) pylint:disable=W0212
-    def setUp(self):  # noqa pylint:disable=C0103
+    # (test names are long to make it readable) pylint:disable=C0103
```

Comment:
Thats an option.  I used the existing strategy in this module.  This is something which needs fixing in entire robottelo if anybody has free cycles, I do not at this time.